apps/src/lab2/views/dialogs/generic-dialog.module.scss

```diff
@@ -33,36 +33,4 @@
       justify-content: flex-end;
     }
   }
-
```

Comment:
I removed these styles since they conflict or are redundant with the styling baked into the DSCO Button component.